deploy/docs/Installation_with_Helm.md

```diff
@@ -104,6 +104,7 @@ The following command will install the Sumo Logic chart with the release name
 
 ```bash
 helm upgrade --install my-release sumologic/sumologic \
+  --version=2.0 \
```

Comment:
Should this change live in `main` branch or only  in `release-v2.0` branch?

Setting this on `main` means we need to update it when we release `v2.1` and so forth.

Reply:
Will this install the latest 2.0 every time? WE will need to keep this updated with every release right?